roles/platform/tasks/initialize_teardown.yml

```diff
@@ -15,13 +15,15 @@
 # limitations under the License.
 
 - name: Refresh Environment Info with Descendants
+  when: no
```

Comment:
I would rather not change this in multiple places with this 'no' approach, it would be better to simply not run that collection in the cloud module so the change is in a single place and easy to rollback when the bug is fixed.